commands.go

```diff
@@ -246,17 +262,24 @@ func CommandModify(conf Config, ctx, cmdLine CmdLine) error {
 // CommandNext prints the unresolved tasks associated with the current context.
 // This is the default command.
 func CommandNext(conf Config, ctx, cmdLine CmdLine) error {
+
 	ts, err := NewTaskSet(
 		conf.Repo, conf.IDsFile, conf.StateFile,
 		WithoutStatuses(STATUS_TEMPLATE),
 		WithStatuses(NON_RESOLVED_STATUSES...),
+		WithIDs(cmdLine.IDs...),
+		WithProjects(ctx.Project, cmdLine.Project),
+		WithoutProjects(ctx.AntiProjects...),
+		WithoutProjects(cmdLine.AntiProjects...),
+		WithTags(ctx.Tags...),
+		WithTags(cmdLine.Tags...),
+		WithoutTags(ctx.AntiTags...),
+		WithoutTags(cmdLine.AntiTags...),
+		WithText(cmdLine.Text),
```

Comment:
We pass command line Text into the constructor as well, since it can technically also act as a filter in the context of the `next` command.